WRITING.md

```diff
@@ -142,6 +142,17 @@ Default: `false`
 <Since v="0.24.0" />
 ```
 
+### Version Component
+
+Sometimes it may be useful to display the latest version of a package in a page. For this, you can use the `<Version />` component, which takes the `pkgName` property that must receive a valid package name in the npm registry as value.
```

Comment:
A bit of a language nit, but I hope this is a tiny bit clearer.

```suggestion
Sometimes it may be useful to display the latest version of a package on a page. For this, you can use the `<Version />` component, which must receive a valid package name from the npm registry as its `pkgName` prop.
```